corehq/apps/commtrack/static/commtrack/ko/locations.js

```diff
@@ -6,6 +6,10 @@ $(function() {
 
     model.load(settings);
     ko.applyBindings(model, $('#settings').get(0));
+
+    $("form#settings").on("change input", function() {
```

Comment:
http://api.jquery.com/on/

Docs about the first argument of the `.on` function:

```
events
Type: String
One or more space-separated event types and optional namespaces, such as "click" or "keydown.myPlugin".
```

The second parameter in your example is actually a selector, not an event. That lets you wrap the event on something higher in the page (usually because the thing you're trying to manipulate isn't loaded yet).